supply-chain-security/compromises/README.md

```diff
@@ -37,6 +48,12 @@ We welcome additions to this catalog by
 | [Ceph and Inktank](2015/ceph-and-inktank.md) | 2015 | Build, source and publishing infrastructure | [1](https://www.zdnet.com/article/red-hats-ceph-and-inktank-code-repositories-were-cracked/) |
 | [Code Spaces](2014/code-spaces.md) | 2014    | Source Code Compromise| [1](https://threatpost.com/hacker-puts-hosting-service-code-spaces-out-of-business/106761/)
 | [Monju Incident](2014/monju.md) | 2014    | Publishing infrastructure| [1](https://www.contextis.com/en/blog/context-threat-intelligence-the-monju-incident)
+| [APT lack of validation for source packages](2013/apt.md) | 2013 | Build infrastructure | [1](https://lwn.net/Articles/602461/) |
+| [kernel.org compromise](2011/kernelorg.md) | 2011 | Publishing infrastructure | [1](https://lwn.net/Articles/461237/), [2](https://lwn.net/Articles/461552/) |
+| [apache.org incident](2010/apache.md) | 2010 | Build and publishing infrastructure | [1](https://blogs.apache.org/infra/entry/apache_org_04_09_2010) |
 | [Operation Aurora](2010/aurora.md) | 2010 | Watering-hole attack | [1](https://www.wired.com/2010/03/source-code-hacks/) | 
 | [ProFTPD](2010/proftpd.md) | 2010 | Source Code Repository | [1](https://www.zdnet.com/article/open-source-proftpd-hacked-backdoor-planted-in-source-code/) |
+| [WordPress backdoor](2007/wordpress.md) | 2007 | Source Code Repository | [1](https://lwn.net/Articles/224997/) |
```

Comment:
remove `(Repository)` for consistency with definitions and add `(Publishing Infrastructure)` as a new line